cmd/temporalite/main.go

```diff
@@ -35,18 +37,20 @@ var (
 )
 
 const (
-	ephemeralFlag   = "ephemeral"
-	dbPathFlag      = "filename"
-	portFlag        = "port"
-	metricsPortFlag = "metrics-port"
-	uiPortFlag      = "ui-port"
-	headlessFlag    = "headless"
-	ipFlag          = "ip"
-	logFormatFlag   = "log-format"
-	logLevelFlag    = "log-level"
-	namespaceFlag   = "namespace"
-	pragmaFlag      = "sqlite-pragma"
-	configFlag      = "config"
+	ephemeralFlag            = "ephemeral"
+	dbPathFlag               = "filename"
+	portFlag                 = "port"
+	metricsPortFlag          = "metrics-port"
+	uiPortFlag               = "ui-port"
+	headlessFlag             = "headless"
+	ipFlag                   = "ip"
+	logFormatFlag            = "log-format"
+	logLevelFlag             = "log-level"
+	namespaceFlag            = "namespace"
+	pragmaFlag               = "sqlite-pragma"
+	configFlag               = "config"
+	dynamicConfigValueFlag   = "dynamic-config-value"
+	searchAttributeCacheFlag = "search-attribute-cache"
```

Comment:
Gonna probably remove this and instead set a default of true if they don't pass `--dynamic-config-value system.forceSearchAttributesCacheRefreshOnRead=false`.